### DISCUSSION TITLE
New AR-js-org welcome page

### DISCUSSION THREAD

**Discussion by kalwalt** [AR-js-org/AR.js#534] [Ideas]
It's possible to create an entry page for the organization as explaned in [the github docs](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) I think that is possible also post some link of AR.js examples, we can store the examples there or in another place. Actually there are many issues with the AR.js examples hosted on gh-pages, and we can solve brillantly the problem.
I will create a simple one to start but if someone want to help is very welcome.

**Comment by kalwalt** [comment]
We have the [profile page](https://ar-js-org.github.io/.github/profile) and i hosted some examples for image tracking and marker tracking. @nickw1 I have not included a location based example, was not sure what to show, if you suggest one i will add to it. I would keep this page simple as much possible, we can add some more infos and in a near future, it could became our official Website.

**Comment by nickw1** [comment]
@kalwalt looks good! I was thinking of the example which obtains your location, then creates 4 boxes to the north, south, east and west of you. If you're on a mobile device you can then walk towards the boxes. i.e perhaps this example? https://github.com/AR-js-org/AR.js/blob/master/aframe/examples/new-location-based/basic-js/

**Reply by kalwalt** [reply]
> @kalwalt looks good! I was thinking of the example which obtains your location, then creates 4 boxes to the north, south, east and west of you. If you're on a mobile device you can then walk towards the boxes. i.e perhaps this example? https://github.com/AR-js-org/AR.js/blob/master/aframe/examples/new-location-based/basic-js/

Yes this is the example that we can show as a live example, others require a special setup or are more complex. The idea is only to show few simple examples, I vote for this.

**Reply by nickw1** [reply]
Sounds good, are you happy to upload it or shall I?

**Reply by kalwalt** [reply]
I am happy to upload it, but i will appreciate it if you can do it. :slightly_smiling_face:

**Reply by nickw1** [reply]
OK will do.
